docs/modders/Random_Map_Template.md

```diff
@@ -59,7 +60,7 @@
 ``` javascript
 {
 	// Type of this zone. Possible values are:
-	// "playerStart", "cpuStart", "treasure", "junction"
+	// "playerStart", "cpuStart", "treasure", "junction", "sealed"
```

Comment:
Perhaps more detailed explanation for these zone types is in order?
```suggestion
	// "playerStart" - this zone is used as starting zone for a "human or CPU" players
	//	"cpuStart" - this zone is used as starting zone for "CPU only" players
	//	"treasure" - generic neutral zone
	//	"junction" - small neutral zone that connects two or more other zones
	//	"sealed" - decorative empty zone that is filled with obstacles
```
(feel free to replace with better descriptions)